### ISSUE TITLE
Symmetrically normalize adjacency matrix

### ISSUE THREAD

**Issue by XiaoYee** [tkipf/pygcn#8]
Why in this pytorch implement，use row-normalize sparse matrix both for features and adj which is different from tensorflow implement？
